ci/deploy_pypi.sh

```diff
@@ -11,4 +11,4 @@ if [ -d $TRAVIS_BUILD_DIR/build/wheelhouse ]; then
       twine upload --username pairinteraction-slave --password $PYPI_TOKEN --skip-existing --repository-url https://test.pypi.org/legacy/ $TRAVIS_BUILD_DIR/build/wheelhouse/*.whl;
       ;;
   esac;
-fi
\ No newline at end of file
+fi
```

Comment:
This file is unused since moving to GitHub Actions. Maybe we can just delete it?